sqlitedict.py

```diff
@@ -105,6 +105,18 @@ def decode(obj):
     return loads(bytes(obj))
 
 
+def print_tablenames(filename):
```

Comment:
IMO better to move it to SqliteDict class and use as method, wdyt @transfluxus ?